src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ISymbolExtensions.cs

```diff
@@ -106,6 +106,9 @@ public static ImmutableArray<ISymbol> ExplicitInterfaceImplementations(this ISym
 
         public static ImmutableArray<ISymbol> ExplicitOrImplicitInterfaceImplementations(this ISymbol symbol)
         {
+            if (symbol.Kind != SymbolKind.Method && symbol.Kind != SymbolKind.Property && symbol.Kind != SymbolKind.Event)
```

Comment:
I see some of the callsites also have this exact same check - might be worth auditing the callsites and removing duplicate kind checks now: https://github.com/dotnet/roslyn/blob/b28f5fec9aba8b9727f0937a3afebc02be7ed04c/src/Workspaces/Core/Portable/Shared/Extensions/ISymbolExtensions.cs#L290-L293